.yamllint.yaml

```diff
@@ -0,0 +1,4 @@
+extends: default
+
+rules:
+  line-length: disable
```

Comment:
Also, is there more that we can do? I saw this list of rules on some random repo, which may be a bit excessive, but some of them we might want to do:

```
rules:
  braces:
    max-spaces-inside: 1
  comments:
    level: error
  comments-indentation:
    level: warning
  document-end:
    present: no
  document-start:
    level: error
    present: no
  empty-lines:
    max: 1
    max-start: 0
    max-end: 0
  line-length:
    level: warning
    max: 120
```